docs/en/guides/20-cloud/10-using-databend-cloud/06-aws-iam-role.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Access your own AWS S3 bucket in Databend Cloud with IAM Role"
+---
+
+# Why IAM Role
+
+With AWS IAM Role, you can access your own AWS S3 buckets in Databend Cloud. This allows you to securely access your data and perform data analysis without having to manage your AWS credentials.
+
+# How to Use IAM Role
+
+1. Get the IAM role ARN for your Databend Cloud organization:
```

Comment:
how to get this role ARN in databend cloud?

Reply:
We could add it to web interface later, but now only by ticket.